README.md

```diff
@@ -11,218 +11,14 @@
 Mutations are persisted when Offline
 and replicated back to server when online.
 
+## Documentation
 
+Link to be added.
```

Comment:
I can register domain. android.offix.dev is ok?

Reply:
Yep, perfect! ❤️